src/SignalR/common/SignalR.Common/src/Protocol/HandshakeResponseMessage.cs

```diff
@@ -18,34 +18,13 @@ public class HandshakeResponseMessage : HubMessage
         /// </summary>
         public string Error { get; }
 
-        /// <summary>
-        /// Highest minor protocol version that the server supports.
-        /// </summary>
-        public int MinorVersion { get; }
```

Comment:
We should include the ProtocolVersion in the handshake response now IMHO. We could add it later, but it might be quicker and easier to to just replace MinorVersion with ProtocolVersion instead of making that two separate changes.

Reply:
I get your point but along that line of logic we should just leave the Minor protocol version though, right? If we don't plan to use it in the handshake response now then I think we might as well leave it out.

Reply:
I don't feel too strongly about it. I'm fine just removing it for now.